src/components/search/facets/facet-selected-filters.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <div v-if="selectedFilters.length" class="x-ellipsis">
+  <div v-if="selectedFilters.length">
     <span
       v-if="selectedFilters.length >= 0"
```

Comment:
Do you think that we can now merge these 2 nodes into one? The `div` is adding nothing now. I can see no difference by merging them.